CONTRIBUTING.md

```diff
@@ -196,39 +195,56 @@ existing_file
 my_new_file
 ```
 
-#### Adding external links
-If you are adding references to an external link (e.g. `https://github.com/neuroinformatics-unit/movement/issues/1`) in a `.md` file, you will need to check if a matching URL scheme (e.g. `https://github.com/neuroinformatics-unit/movement/`) is defined in `myst_url_schemes` in `docs/source/conf.py`. If it is, the following `[](scheme:loc)` syntax will be converted to the [full URL](movement-github:issues/1) during the build process:
+#### Linking to external URLs
+If you are adding references to an external URL (e.g. `https://github.com/neuroinformatics-unit/movement/issues/1`) in a `.md` file, you will need to check if a matching URL scheme (e.g. `https://github.com/neuroinformatics-unit/movement/`) is defined in `myst_url_schemes` in `docs/source/conf.py`. If it is, the following `[](scheme:loc)` syntax will be converted to the [full URL](movement-github:issues/1) during the build process:
 ```markdown
 [link text](movement-github:issues/1)
 ```
 
-If it is not yet defined and you have multiple external links pointing to the same base URL, you will need to [add the URL scheme](myst-parser:syntax/cross-referencing.html#customising-external-url-resolution) to `myst_url_schemes` in `docs/source/conf.py`.
-
+If it is not yet defined and you have multiple external URLs pointing to the same base URL, you will need to [add the URL scheme](myst-parser:syntax/cross-referencing.html#customising-external-url-resolution) to `myst_url_schemes` in `docs/source/conf.py`.
 
 ### Updating the API reference
-The API reference is auto-generated by the `docs/make_api_index.py` script, and the `sphinx-autodoc` and `sphinx-autosummary` plugins.
+The API reference is auto-generated by the `docs/make_api_index.py` script, and the [sphinx-autodoc](sphinx-doc:extensions/autodoc.html) and [sphinx-autosummary](sphinx-doc:extensions/autosummary.html) extensions.
 The script generates the `docs/source/api_index.rst` file containing the list of modules to be included in the [API reference](target-api).
 The plugins then generate the API reference pages for each module listed in `api_index.rst`, based on the docstrings in the source code.
 So make sure that all your public functions/classes/methods have valid docstrings following the [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) style.
 Our `pre-commit` hooks include some checks (`ruff` rules) that ensure the docstrings are formatted consistently.
 
 If your PR introduces new modules that should *not* be documented in the [API reference](target-api), or if there are changes to existing modules that necessitate their removal from the documentation, make sure to update the `exclude_modules` list within the `docs/make_api_index.py` script to reflect these exclusions.
 
+#### Cross-referencing Python objects
```

Comment:
I think this section should be a level-3 heading. That's because cross-references are used not only for the API reference (i.e. in docstrings) but also in examples and in other doc pages (e.g. "The movement datasets").

In docstrings and examples, the rst syntax is used (the one you've documented here), but in .md files we use the equivalent myst-markdown syntax, e.g. ``{class}`xarray.Dataset` ``. This should also bo documented.

Reply:
Thanks @niksirbi! I've moved the entire section and placed it after "Updating the examples" (given it's kind of a "more advanced" topic) and before "Building the docs locally" (so the section doesn't go unnoticed). I've also attempted to separate internal and external references. Re-requesting your review to get your input on these changes.